Fix grammar and punctuation in dev tunnel login section (#1485)
MicrosoftDocs/azure-dev-docs: articles/dev-tunnels/get-started.md

## Login

> To start a dev tunnel, you first need to log in with either a Microsoft Entra ID, Microsoft, or GitHub account. Dev tunnels doesn't support hosting tunnels anonymously. For more information take a look at the [CLI command reference](cli-commands.md) documentation.

Reviewer: ```suggestion
To start a dev tunnel, you first need to log in with either a Microsoft Entra ID, Microsoft, or GitHub account. Dev tunnels doesn't support hosting tunnels anonymously. For more information, see [CLI command reference](cli-commands.md) documentation.
```
